docusaurus/docs/Android/03-ui/02-general-customization/02-chatui.mdx

```diff
@@ -149,11 +149,6 @@ ChatUI.avatarBitmapFactory = object : AvatarBitmapFactory(context) {
         val imageResult = context.imageLoader.execute(
             ImageRequest.Builder(context)
                 .data(user.image)
-                .apply {
-                    if (!user.online) {
-                        transformations(BlurTransformation(context))
-                    }
-                }
```

Comment:
`BlurTransformation` has been permanently deprecated.

Reply:
What do you think about guiding copy & paste the [BlurTransformation](https://github.com/coil-kt/coil/blob/845f39383f332428077c666e3567b954675ce248/coil-base/src/main/java/coil/transform/BlurTransformation.kt) to users who use Coil 2.x?

From the Coil docs:
> [BlurTransformation](https://github.com/coil-kt/coil/blob/845f39383f332428077c666e3567b954675ce248/coil-base/src/main/java/coil/transform/BlurTransformation.kt) and [GrayscaleTransformation](https://github.com/coil-kt/coil/blob/845f39383f332428077c666e3567b954675ce248/coil-base/src/main/java/coil/transform/GrayscaleTransformation.kt) are removed from the library. If you use them, you can copy their code into your project.

Reply:
Updated with [GrayScaleTransformation](https://github.com/GetStream/stream-chat-android/pull/3538/commits/bba2c8897878d81f3e7f033856a84bd6398f984e). docs: [9d1d434](https://github.com/GetStream/stream-chat-android/pull/3538/commits/9d1d4342835b2841efd99f3db30ad1d3e90b2145).